base_import_security_group/i18n/base_import_security_group.pot

```diff
@@ -0,0 +1,25 @@
+# Translation of Odoo Server.
```

Comment:
you can drop this file

Reply:
@i-vyshnevska : I have remove `base_import_security_group.pot` file